app/controllers/hiring_staff/sign_in/email/sessions_controller.rb

```diff
@@ -35,12 +35,14 @@ def change_organisation
   end
 
   def choose_organisation
-    # This flow needs to be adapted when we have implemented school groups/trusts/LAs
-    # It is likely that we will not sign in Trust users as a school.
     information = GetInformationFromLoginKey.new(get_key)
-    @reason_for_failing_sign_in, @schools = information.reason_for_failing_sign_in, information.schools
+    @reason_for_failing_sign_in = information.reason_for_failing_sign_in
+    @schools, @school_groups = information.schools, information.school_groups
     update_session_without_urn(information.details_to_update_in_session)
-    redirect_to auth_email_create_session_path(urn: @schools.first.urn) if @schools&.one?
+    # .to_i used to convert nil to 0
+    only_one_organisation = @schools&.count.to_i + @school_groups&.count.to_i == 1
```

Comment:
A minor thing, but I like the Ruby convention where names for boolean variables end with a `?` - makes it easy to see what it's for.